### DISCUSSION TITLE
Problem while running pytest with pyargs in local venv

### DISCUSSION THREAD

**Discussion by JackyZzZz** [theochem/Selector#206] [Coding]
It looks like pyargs are causing some issues when pytest-xdist distributes tests across workers.

![image](https://github.com/theochem/Selector/assets/110846941/2d4b0dfc-b9a1-4ef9-af19-94ee253ca54b)

However, ```pytest selector -n auto``` seems to work for parallel execution of tests. I am trying to understand what is happening under the hood, but I haven't find an answer yet. Is there some other set up we need to do, or is this problem caused by pyargs maybe? If it is, then I think we should update the documentation because it's kind of confusing from there.

**Comment by FanwangM** [comment]
Nice catch! @JackyZzZz 

I will read this later today or tomorrow. @FanwangM
